### DISCUSSION TITLE
Where is the bonus xp for shadow pokemon stored?

### DISCUSSION THREAD

**Discussion by lukeparkerprog** [kwsch/PKHeX#3695] [Q&A]
Hi, in pokemon colosseum/xd shadow pokemon store all the experience they gain and are granted it all at once when purified. I was just wondering where this number can be seen and changed.

**Comment by kwsch** [comment]
It's stored in one of the shadow monitor structures, not thoroughly documented enough to have a GUI. Since it's more of a cosmetic edit (EXP of an entity can be changed after purification), it's not really a priority to investigate.

https://github.com/kwsch/PKHeX/blob/a8be90be315364fbf4609c6ee5f20a4b6a57a3eb/PKHeX.Core/Saves/Substructures/Gen3/CXD/ShadowInfoEntryXD.cs#L28-L32

https://github.com/kwsch/PKHeX/blob/a8be90be315364fbf4609c6ee5f20a4b6a57a3eb/PKHeX.Core/Saves/Substructures/Gen3/CXD/ShadowInfoEntryColo.cs
